protocols/directdefund/directdefund.go

```diff
@@ -37,10 +37,10 @@ const (
 
 // Objective is a cache of data computed by reading from the store. It stores (potentially) infinite data
 type Objective struct {
-	Status               protocols.ObjectiveStatus
-	C                    *channel.Channel
-	finalTurnNum         uint64
-	transactionSubmitted bool // whether a transition for the objective has been submitted or not
+	Status                       protocols.ObjectiveStatus
+	C                            *channel.Channel
+	finalTurnNum                 uint64
+	withdrawTransactionSubmitted bool // whether a withdraw transaction has been declared as a side effect in a previous crank
```

Comment:
nit:

```suggestion
	// whether a withdraw transaction has been declared as a side effect in a previous crank
	withdrawTransactionSubmitted bool
```

makes this commentary visible at dev time as godoc.